src/roles/vsc-deploy/tasks/setup_vsc_config.yml

```diff
@@ -85,7 +89,38 @@
       {{ proxy_conf | default('') }} {{ tmp_dir_name }}/{{ config_file_name }}
       {{ vsc_username|default(vsc_default_username) }}@{{ '[' }}{{ mgmt_ip }}{{ ']' }}:
     no_log: "{{ lookup('env', 'METROAE_NO_LOG') or 'true' }}"
-    when: mgmt_ip | ipv6
+    when:
+      - mgmt_ip | ipv6
+      - not override_vsc_config
+
+  - block:
+    - name: Copy user provided config file for VSC(IPv4)
+      connection: local
+      shell: >-
+        sshpass -p{{ vsc_password | default(vsc_default_password) }} scp -o StrictHostKeyChecking=no -o UserKnownHostsFile=/dev/null
```

Comment:
You will need single quotes around vsc_password.  Urvish just put in this change this week.

Reply:
added